etlman/projects/models.py

```diff
@@ -10,7 +10,6 @@
     IntervalSchedule,
     PeriodicTask,
 )
-from simple_history.models import HistoricalRecords
```

Comment:
It looks like there might be some accidental deletions in this PR?

Reply:
TBH... I am unsure how that happened 😅. It should be back now.